sdk/tdf.go

```diff
@@ -718,6 +718,26 @@ func (r *Reader) DataAttributes() ([]string, error) {
 	return attributes, nil
 }
 
+/*
+*
+Retrieve the payload key, either from performing an unwrap or from a previous unwrap,
+and write it to a user buffer.
+
+OUTPUTS:
+  - []byte - Byte array containing the DEK.
+  - error - If an error occurred while processing
+*/
+func (r *Reader) PayloadKey() ([]byte, error) {
```

Comment:
```suggestion
/*
*WARNING:* Using this function is unsafe since KAS will no longer be able to prevent access to the key.

Retrieve the payload key, either from performing an unwrap or from a previous unwrap,
and write it to a user buffer.

OUTPUTS:
  - []byte - Byte array containing the DEK.
  - error - If an error occurred while processing
*/
func (r *Reader) UnsafePayloadKeyRetrieval() ([]byte, error) {
```

Reply:
Why would it be unsafe? KAS has to allow access to the key before this function can be used.

Reply:
Agreed, this should be properly marked. I have committed your suggestions.